tools/downloader/src/open_model_zoo/model_tools/_common.py

```diff
@@ -110,3 +128,44 @@ def get_package_path(python_executable, package_name):
     # For a package, the file is __init__.py, so to get the package path,
     # take its parent directory.
     return file_path.parent
+
+def get_version():
+    get_ie_version = subprocess.run(
+        [str(sys.executable), '-c',
+            'from openvino.inference_engine import ie_api;'
+                'print(ie_api.get_version())',
+        ],
+        stdin=subprocess.DEVNULL,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.DEVNULL,
+        universal_newlines=True,
+    )
+
+    if get_ie_version.returncode == 0:
+        version = get_ie_version.stdout.rstrip('\n')
+        return version
+    else:
+        try:
+            commit_hash = subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip().decode()
+            return f'custom_{commit_hash}'
+        except Exception:
+            return 'unknown version'
```

Comment:
AFAIK, the current plan is to only enable telemetry when packaged inside the OpenVINO toolkit, which means that this section is unneeded.

(That said, in case something like this is re-added in the future: make sure to point `git` to the correct repository. By default, it will use the working directory, which could be anything.)

Reply:
Done